symphony-bdk-http/symphony-bdk-http-api/src/main/java/com/symphony/bdk/http/api/Pair.java

```diff
@@ -3,38 +3,58 @@
 import lombok.Getter;
 import org.apiguardian.api.API;
 
+import java.util.ArrayList;
+import java.util.List;
+
 /**
  * Pair of string values. Used by generated code only.
  */
 @Getter
 @API(status = API.Status.INTERNAL)
 public class Pair {
 
-    private String name = "";
-    private String value = "";
+  private String name = "";
+  private String value = "";
 
-    public Pair(String name, String value) {
-        this.setName(name);
-        this.setValue(value);
-    }
+  public Pair(String name, String value) {
+    this.setName(name);
+    this.setValue(value);
+  }
 
-    private void setName(String name) {
-        if (isInvalidString(name)) {
-            return;
-        }
+  public static Pair pair(String name, String value) {
+    return new Pair(name, value);
+  }
 
-        this.name = name;
+  public static List<Pair> pairs(String... pairs) {
```

Comment:
I would probably let the developer build a list of pairs with List.of(pair(1,2), pair(3,4)) to avoid runtime errors and detect problems at compile time